daycare/migrations/0005_image_data_migration.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 4.0.4 on 2022-04-11 19:07
+
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('daycare', '0004_static_daycare_users'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from daycare.models import DayCare, Image
+
+        DAYCARE_IMAGE_LIST = [
+            "https://i.pinimg.com/originals/83/c8/56/83c856787dec2ebb9fec486242f9e1d6.jpg",
+            "https://reddogpetresort.com/wp-content/uploads/2020/05/IMG_1701.jpg",
+            "https://www.crowdfunder.co.uk/uploads/projects/26396.jpg",
+            "http://savvyvirtualassistant.com/BFBB/wp-content/uploads/2012/03/290055_482309965129233_132653009_o.jpg",
+            "https://www.theboneadventure.com/images/bone-backyard/greenhouse.jpg",
+            "https://www.theboneadventure.com/images/bone-home/bone-home04.jpg",
+            "https://www.theboneadventure.com/images/600x350-Image-OliveBranch.jpg",
+            "https://www.thepoochpatio.com/images/home-featured.jpg",
+            "https://boston.cbslocal.com/wp-content/uploads/sites/3859903/2014/01/commondog.jpg",
+            "https://i.pinimg.com/originals/51/47/2f/51472f3314975b24b58e999c8878a0b1.jpg",
+            "https://i.pinimg.com/originals/36/c0/a9/36c0a940cc5904337eb949a6634bccf3.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-1.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-2.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-3.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-4.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-5.jpg"
+        ]
+
+        static_daycare_id = []
+        static_daycare_images = []
+
+        for daycare in DayCare.objects.all():
+            if 'static' in daycare.user.username:
+                static_daycare_id.append(daycare)
+
+        for index, daycare in enumerate(static_daycare_id):
+            for i in range(5):
+                static_daycare_images.append((daycare, DAYCARE_IMAGE_LIST[(index + i) % len(DAYCARE_IMAGE_LIST)]))
```

Comment:
Sorry, but I don't understand what is happening here or why (why are we calculating stuff?)

Reply:
instead of random, every static daycare user will get 5 images, so that each image will be selected by the daycare index combined with the image index, and in order not to exceed the size of the image set, I added a modulo.
so that daycare01 will get images 1,2,...,5, daycare2 will get images 2,3...,6, and so on.
actually, I implemented the explicit data in the previous push, to run in a loop as you commented.

do you recommend any other effective solution instead of random, this way, or the last method I used with explicit?

Reply:
At this point, things are over-complicated. the fact that we need to think over a trivial addition like this for so long means there is a design flaw. 
If we're set on this "static daycare" implementation, I'd explicitly have photo X attached to daycare Y. whether it means having the same photo for multiple daycares, more photos, or less daycares, is up to you. but it needs to be clear and explicit enough so "my grandmother will be able to understand it".

Reply:
I decided to make the things more simple because its only static data so it doesn't really matter which daycare will get each image, so every daycare will get all the images in the static images list by using
`list comprehension`.

Reply:
Never say never. even though it is static data, it does matter. 
I'm okay with the current implementation, while having all the photos is less reliable than having specific ones mentioned, it is a good-enough compromise and it is explicit enough.

The list comprehension is also well done! 
I'm not sure the second one is absolutely needed, but it is good enough as is

good job!